doc/src/custom/make-index.sh

```diff
@@ -20,20 +20,16 @@
 
```

Comment:
I think we can ditch this script as well, Sphinx can take care of handling multiple versions of the build docs.

Reply:
[Addressing in latest comment; see :high_brightness:.]